docs/references/qa/CometBFT-QA-v1.md

```diff
@@ -0,0 +1,307 @@
+summarizes the values, i.e., the number of transaction processed, plotted in the figure. We can see the saturation point in the diagonal defined
+by `c=1,r=400` and `c=2,r=200`:
+
+| r    | c=1       | c=2       | c=4   |
+| ---: | --------: | --------: | ----: |
+| 200  | 17800     | **34600** | 50464 |
+| 400  | **31200** | 54706     | 49463 |
+| 800  | 51146     | 51917     | 41376 |
+| 1600 | 50889     | 47732     | 45530 |
+
+For comparison, this is the table obtained on the baseline version, with the same saturation point:
+
+| r    | c=1       | c=2       | c=4   |
+| ---: | --------: | --------: | ----: |
+| 200  | 17800     | **33259** | 33259 |
+| 400  | **35600** | 41565     | 41384 |
+| 800  | 36831     | 38686     | 40816 |
+| 1600 | 40600     | 45034     | 39830 |
+
+In conclusion, we chose `c=1,r=400` as the transaction load that we will use in the rest of QA
+process. This is the same value used in the previous QA tests.
+
+### With latency emulation
+
+For comparing where the network starts to saturate in `v1` with and without latency emulation, we
+run a new set of experiments with different configurations of transaction loads: we use only one
+connection and a transaction rate ranging from 100 to 1000 tx/s, in intervals of 100. The figure
+depicts in total six instances of these experiments, three with latency emulation and three without. 
+
+![v1_saturation](imgs/v1/saturation/saturation_v1_LE.png) 
+
+Up to 300 tx/s, the throughput is optimal for both configurations. However, when the load increases
+beyond this threshold, not all transactions are processed. Given the limited number of experiments
+conducted, it's challenging determining conclusively which configuration offers better throughput.
+Nevertheless, we can still say that there are no big discrepancies in the obtained values on both
+scenarios.
+
+## 200-nodes test
+
+This experiment consist in running the 200-nodes network, injecting a load of 400 tx/s (`c=1,r=400`)
+during 90 seconds, and collecting the metrics. The network composition is the same as used for
+finding the saturation point.
+
+For the experiments with latency emulation have set a duration of 180 seconds instead of the 90.
+
+### Latencies
+
+The following figures show the latencies of the experiment. Each dot represents a block: at which
+time it was created (x axis) and the average latency of its transactions (y axis).
+
+| v0.38 | v1 (without LE / with LE) 
+|:--------------:|:--------------:|
+| ![latency-1-400-v38](img38/200nodes/e_de676ecf-038e-443f-a26a-27915f29e312.png) | ![latency-1-400-v1](imgs/v1/200nodes/latencies/e_8e4e1e81-c171-4879-b86f-bce96ee2e861.png) 
+| | ![latency-1-400-v1-le](imgs/v1/200nodes_with_latency_emulation/latencies/e_8190e83a-9135-444b-92fb-4efaeaaf2b52.png)
+
+In both cases, most latencies are around or below 4 seconds. On `v0.38` there are peaks reaching 10
+seconds, while on `v1` (without LE) the only peak reaches 8 seconds. In general, the images are
```

Comment:
You might want to mention that `v0.38.x` didn't have PBTS, whereas `v1.x` does have it (IIUC, you activated PBTS in the 200-node manifest).
The implication is that PBTS tends to produce slightly smaller latencies:
* in PBTS, the block timestamp is the proposer's wallclock value when the block was created
* for reference, with BFT-time (pre-PBTS), the block's timestamp is the median of the timestamp of all precommits in the previous height. Each of those timestamps is the validator's wallclock time when the precommit is sent

So, latencies calculated when BFT-Time is active tend to contain one extra network propagation delay (the one whereby precommits are disseminated).

Reply:
That's a good point! I added it to the text.